libs/ledger-live-common/src/families/celo/js-signOperation.ts

```diff
@@ -9,15 +9,32 @@ import buildTransaction from "./js-buildTransaction";
 import { rlpEncodedTx, encodeTransaction } from "@celo/wallet-base";
 import { tokenInfoByAddressAndChainId } from "@celo/wallet-ledger/lib/tokens";
 import { withDevice } from "../../hw/deviceAccess";
+import { OperationType } from "../../types";
+
+const MODE_TO_TYPE = {
```

Comment:
We could get some extra type safety here by setting MODE_TO_TYPE to:

```
const MODE_TO_TYPE: { [key: CeloOperationMode]: string } = 
```

You'd have to remove `default` as a property - maybe keep that as a separate var.  But that way, we'd be able to ensure that a type always exists for each mode in this file.